pkg/ansible/runner.go

```diff
@@ -14,19 +14,25 @@ import (
 )
 
 const (
-	operatorNamespace            = "cluster-operator"
-	openshiftAnsibleContainerDir = "/usr/share/ansible/openshift-ansible/"
-	sshKeySecretName             = "ssh-private-key"
-	awsCredsSecretName           = "aws-credentials"
+	operatorNamespace             = "cluster-operator"
+	openshiftAnsibleContainerDir  = "/usr/share/ansible/openshift-ansible/"
+	sshKeySecretName              = "ssh-private-key"
+	awsCredsSecretName            = "aws-credentials"
+	DefaultAnsibleImage           = "openshift/origin-ansible:v3.8"
```

Comment:
These two consts should be removed. I see that they are used in infra_controller_test.go. However, different dummy values can be used there. Having them here gives the impression that they are somehow useful to the production code.